sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventDirectoryPage.java

```diff
@@ -22,4 +22,35 @@
 
 public class EventDirectoryPage {
   public static final By NEW_EVENT_BUTTON = By.id("eventNewEvent");
+  public static final By SEARCH_SPECIFIC_EVENT_BUTTON = By.id("eventSearchSpecificEvent");
+  public static final By SORT_BY_ALL = By.id("all");
+  public static final By SORT_BY_SIGNAL = By.id("status-Signal");
+  public static final By SORT_BY_EVENT = By.id("status-Event");
+  public static final By SORT_BY_SCREENING = By.id("status-Screening");
+  public static final By SORT_BY_CLUSTER = By.id("status-Cluster");
+  public static final By SORT_BY_DROPPED = By.id("status-Dropped");
+  public static final By SEARCH_EVENT_BY_FREE_TEXT = By.id("freeText");
+  public static final By RESET_FILTER = By.id("actionResetFilters");
+  public static final By APPLY_FILTER = By.id("actionApplyFilters");
+
+  public static By getByEventUuid(String eventUuid) {
+    return By.cssSelector("a[title='" + eventUuid + "']");
+  }
+
+  public static By getByEventStatus(String eventStatus) {
```

Comment:
We will try to not introduce logic into page objects, for these buttons we will with the corresponding selectors and after we will click by text and for `All` button we will need to implement a method which will click by containing text